app/views/shared/_profile_card_content.html.erb

```diff
@@ -19,7 +19,7 @@
 <% if actor.class.name == "User" %>
   <div class="user-metadata-details">
     <ul class="user-metadata-details-inner">
-      <% if actor.email_public && context != "sidebar" %>
+      <% if actor.setting&.display_email_on_profile && context != "sidebar" %>
```

Comment:
This use of `&.` rubs me the wrong way. If we really didn't create a settings model for a user we'd probably like to be alerted to this fact via an exception, not just silently ignore it.